config/webpack.config.prod.js

```diff
@@ -8,7 +8,8 @@ const UglifyJsPlugin = require('uglifyjs-webpack-plugin');
 const MiniCssExtractPlugin = require('mini-css-extract-plugin');
 const ManifestPlugin = require('webpack-manifest-plugin');
 const InterpolateHtmlPlugin = require('react-dev-utils/InterpolateHtmlPlugin');
-const SWPrecacheWebpackPlugin = require('sw-precache-webpack-plugin');
+const CopyPlugin = require("copy-webpack-plugin");
+const workboxPlugin = require("workbox-webpack-plugin");
```

Comment:
Please use single-quotes for consistency :slightly_smiling_face:

Reply:
fixed :)